tools/symbol-extractor/symbol_extractor.ts

```diff
@@ -74,14 +74,14 @@ export class SymbolExtractor {
 
   static diff(actual: Symbol[], expected: string|((Symbol | string)[])): {[name: string]: number} {
     if (typeof expected == 'string') {
-      expected = JSON.parse(expected);
+      expected = JSON.parse(expected) as (Symbol | string)[];
     }
     const diff: {[name: string]: number} = {};
 
     // All symbols in the golden file start out with a count corresponding to the number of symbols
     // with that name. Once they are matched with symbols in the actual output, the count should
     // even out to 0.
-    (expected as (Symbol | string)[]).forEach((nameOrSymbol) => {
+    expected.forEach((nameOrSymbol) => {
```

Comment:
👍 
You can even remove the parens...

```suggestion
    expected.forEach(nameOrSymbol => {
```